packages/adena-extension/src/resources/chains/chains.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "id": "test3",
+    "main": true,
```

Comment:
key `main` is little bit confused with blockchain terms `mainnet`, `testnet`

Reply:
How about 'default'?

Reply:
works for me